src/app/dashboard/views/DiscoveryAddForm/DiscoveryAddForm.tsx

```diff
@@ -181,6 +181,7 @@ const DiscoveryAddForm = ({ discovery, onClose }: Props): JSX.Element => {
   return (
     <FormikForm<DiscoveryAddValues>
       allowUnchanged
+      aria-label="Add discovery form"
```

Comment:
`form` is already expressed in the form element type and generally unnecessary.
```suggestion
      aria-label="Add discovery"
```